src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -185,26 +172,35 @@ public InternalEngine(EngineConfig engineConfig, boolean skipInitialTranslogReco
         logger.trace("created new InternalEngine");
     }
 
+    private Translog openTranslog(EngineConfig engineConfig, IndexWriter writer, boolean createNew) throws IOException {
+        final Translog.TranslogGeneration generation = loadTranslogIdFromCommit(writer);
+        Translog translog;
+        TranslogConfig translogConfig = engineConfig.getTranslogConfig();
+        if (createNew) {
```

Comment:
In the forceNewTranslog case, do we need to remove the old one?  Who uses forceNewTranslog?